clang/test/CIR/Executables/hello.c

```diff
@@ -2,7 +2,7 @@
 // RUN: %t | FileCheck %s
 // REQUIRES: system-linux
 // REQUIRES: target-linux
-int printf(const char *format);
+#include <stdio.h>
```

Comment:
We cannot do that in the tests, you have to add a mocked version of printf, if you grep clang/test you are very likely to find other people who did it, so you can copy it (sometimes it's under an `Input/` subdir)

Reply:
@bcardosolopes seems like these mocks simply add the declaration of the function. So I just replaced this include by the proper `printf` builtin declaration to simplify things.

Reply:
> @bcardosolopes seems like these mocks simply add the declaration of the function. So I just replaced this include by the proper `printf` builtin declaration to simplify things.

Sounds good! Did you double check that it does trigger the builtin codegen path you just added?

Reply:
I did!